src/gui/pedalboard/cables/Cable.cpp

```diff
@@ -104,6 +104,9 @@ void Cable::repaintIfNeeded (bool force)
         cableBounds.setY (cableBounds.getY() - roundToInt (std::ceil (4.0f * minCableThickness)));
         cableBounds.setHeight (cableBounds.getHeight() + roundToInt (std::ceil (8.0f * minCableThickness)));
 
+        cableBounds.setX (cableBounds.getX() - roundToInt (std::ceil (minCableThickness)));
+        cableBounds.setWidth (cableBounds.getWidth() + roundToInt (std::ceil (2.0f * minCableThickness)));
+
```

Comment:
```suggestion
const auto cableBounds = cablePath.getBounds().expanded (std::ceil (minCableThickness), std::ceil (4.0f * minCableThickness).toNearestInt();
```

The review thing won't let me go up to line 103, but I think we could simplify the code like this.

Reply:
Oh yeah good idea do the whole bounds in one.